common/common_helper.sh

```diff
@@ -707,6 +707,10 @@ function get_urls() {
 			export TECHEMPOWER_URL="${KIND_IP}:${TECHEMPOWER_PORT}"
 		fi
 
+  elif [ ${CLUSTER_TYPE} == "local" ]; then
```

Comment:
Is this used in bulk demo? Do we need to document this?

Reply:
just file indentation changed as per Editor UI rule.